pinot-core/src/test/java/org/apache/pinot/core/segment/processing/framework/SegmentMapperTest.java

```diff
@@ -275,7 +275,7 @@ public void segmentMapperTest(String mapperId, SegmentMapperConfig segmentMapper
     SegmentMapperConfig config11 = new SegmentMapperConfig(_pinotSchema,
         new RecordTransformerConfig.Builder().setTransformFunctionsMap(transformFunctionMap).build(),
         new RecordFilterConfig.Builder().setRecordFilterType(RecordFilterFactory.RecordFilterType.FILTER_FUNCTION)
-            .setFilterFunction("Groovy({timeValue != 1597795200000}, timeValue)").build(), Lists.newArrayList(
+            .setFilterFunction("Groovy({timeValue < 1597795200000L|| timeValue >= 1597881600000}, timeValue)").build(), Lists.newArrayList(
```

Comment:
This is due to changing filtering to be before transform.
Moved filtering to before transform, because I realized while testing, that is transform happens before, user has to be aware of resulting values and apply filter accordingly.
I feel filter then transform makes more sense.
If someone wants filter to apply to transformed column, they can include the transformation in the filter function